static/app/views/insights/common/components/insightsTimeSeriesWidget.tsx

```diff
@@ -142,7 +146,7 @@ export function InsightsTimeSeriesWidget(props: InsightsTimeSeriesWidgetProps) {
       }
 
       return new PlottableDataConstructor(delayedTimeSeries, {
-        color: COMMON_COLORS(theme)[plottableName],
+        color: props.colorPalette?.[idx] ?? COMMON_COLORS(theme)[plottableName],
```

Comment:
This is required because color is no longer part of `timeseries` as it was with the old `series` object